api/proto/config/edgeview.proto

```diff
@@ -0,0 +1,56 @@
+// Copyright(c) 2021 Zededa, Inc.
+// All rights reserved.
+
+syntax = "proto3";
+
+package org.lfedge.eve.config;
+option go_package  = "github.com/lf-edge/eve/api/go/config";
+option java_package = "org.lfedge.eve.config";
+
+message EdgeViewConfig {
+    // JWT token for signed info, it contains the dispatcher
+    // endpoint IP:Port, device UUID and expiration time
+    string  token = 1;
+    // dispatcher certificate(s) if it's not well-known CA signed
+    repeated bytes disp_cert_pem = 2;
+    // policy for device access through edge-view
+    DevDebugAccessPolicy dev_policy = 3;
+    // policy access for apps through edge-view
+    AppDebugAccessPolicy app_policy = 4;
+}
+
+// Dev debug policy applicable to edge-view
+message DevDebugAccessPolicy {
+    // device side of edge-view access is allowed or not
+    bool allow_dev = 1;
+    // allow edge-view ssh into device or not, ssh key still need to be
+    // installed separately even the edge-view allows it
+    bool ssh = 2;
```

Comment:
Currently if the user sets debug.enable.ssh it allows ssh from anybody who can reach the device over IP (on any management port, and perhaps also on app-shared ports - I haven't checked).

With edge-view it might be reasonable to allow ssh only view the edge-view tunnel, which isn't something we can express with what we have here. 
So maybe it makes sense to generalize this ssh boolean to a allow_ssh_from array of strings where one special string value would be "EV" (for edge-view), and other strings can be interface names like "eth2". That way the debug.enable.ssh specification of the authorized_keys doesn't always open it up on eth0. (Need some thoughts on how to transition to this model; perhaps making it a "block_ssh_from" arrary of strings so by default it to allow ssh from anywhere once debug.enable.ssh is set, and one can block e.g., "EV" and/or "eth0" by putting them in the array?

Reply:
Well, this 'ssh' expression is inside the 'edge-view' policy configuration, to allow/dis-allow 'ssh' through the edge-view tunnel. Maybe a new configitem, not for general ssh, but only to allow 'edge-view' ssh setup, can be added to explicitly block the intf side access...

Reply:
Let me sketch on how we can have similar policy controls for our existing ssh, keyboard, and console access with the intent of coming up with a merged approach.